po/pt-BR.po

```diff
@@ -7714,6 +7733,11 @@ msgid ""
 "be dereferenced, eliminating use-after-free and other classes of bugs. But, "
 "GC has a runtime cost and is difficult to tune properly."
 msgstr ""
+"Java, Go e Python dependem do coletor de lixo para identificar a memória que "
+"não é mais acessível e descartá-la. Isso garante que qualquer ponteiro possa "
+"ser desreferenciado, eliminando o uso após a liberação e outras classes de "
+"bugs. Mas, o GC tem um custo de tempo de execução e é difícil de ajustar "
```

Comment:
GC as in garbage collector, but if we translate previously to coletor de lixo, GC means nothing